test/inductor/test_torchinductor.py

```diff
@@ -3371,6 +3379,36 @@ def fn(a):
             ],
         )
 
+    def test_upsample_bilinear2d_invalid_output_size(self):
```

Comment:
I don't think this test belongs in test_torchinductor, it should be somewhere in `test_decomp.py` or wherever decompositions are tested?